runelite-client/src/main/java/net/runelite/client/plugins/maxhit/MaxHitConfig.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2018, magic fTail
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.maxhit;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "maxhitplugin",
+	name = "Max Hit Calculator",
+	description = "Configure boosts for Max Hit"
+)
+
+public interface MaxHitConfig extends Config
+{
+	@ConfigItem(
+		position = 1,
+		keyName = "meleePrayer",
+		name = "Melee Prayer",
+		description = "Choose active prayer when calculating max hit"
+	)
+	default meleePrayer meleeprayer()
+	{
+		return meleePrayer.DEFAULT;
+	}
+
+	enum meleePrayer
```

Comment:
What about showing information for this in tooltip for the max hit? Instead of having it configurable in plugin settings.

Reply:
I considered doing it like that, but with the pot and prayer, there would be 8-10 different combinations, which would just be confusing IMO.

Reply:
That's fine imo, the equipment screen should provide detailed overview. And if you want to quickly check all possible combinations on hover it seems better instead of switching dropdowns in sidebar.